src/nupic/ntypes/Collection.hpp

```diff
@@ -26,8 +26,10 @@
 #include <string>
 #include <vector>
 #include <map>
+#include <nupic/types/Serializable.hpp>
 #include <nupic/utils/Log.hpp>
 
+using namespace cereal;
```

Comment:
ditto, "using namespace" should be within namespace, at least for all headers

Reply:
> "using namespace" should be within namespace, at least for all headers

Do not understand this one....cereal is not a sub-namespce of nupic.
Therefore it should not matter if it is inside or outside of the "using namespace".